### PR TITLE
configurable menu time for RichDisplay

### PR DESCRIPTION
**Pull request by pedall**
- makes menu closing time configurable for developers
- fixes  help message code block to be viewed correctly on mobile
- added `jump` and `firstLast` to typedefs

### CODE REVIEW

File: src/lib/util/ReactionHandler.js [hunk 1]
```diff
@@ -62,6 +62,12 @@ class ReactionHandler extends ReactionCollector {
 		this.prompt = this.options.prompt || 'Which page would you like to jump to?';
 
 		/**
+		 * The time until the reaction collector closes automatically
+		 * @type {number}
+		 */
+		this.time = this.options.time || 30000;
```
Comment (kyranet) [inline src/lib/util/ReactionHandler.js]: This should be

```js
this.time === typeof this.options.time === 'number' ? this.options.time : 30000;
```

File: src/commands/System/help.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ module.exports = class extends Command {
 		const helpMessage = [];
 		for (let cat = 0; cat < categories.length; cat++) {
 			helpMessage.push(`**${categories[cat]} Commands**: \`\`\`asciidoc`);
+			helpMessage.push('');
```
Comment (kyranet) [inline src/commands/System/help.js]: Instead of calling the push method twice, you can push two values into the same array in a single call.

**Comment by bdistin** [comment]
@kyranet I believe this pr effects typings. Can you confirm?

**Comment by kyranet** [comment]
It does, I can patch it in a while once I finish something important, give me 2 hours or so to make sure.

**Pull request merged**